hw/syn/tools/dc/run-syn.tcl

```diff
@@ -63,12 +50,12 @@ set PARAMS ""
 remove_design -designs
 sh rm -rf $WORKLIB/*
 
-analyze -define ${DEFINE} -format sv ${SRC}
-elaborate  ${TOP_ENTITY} -parameters ${PARAMS}
-link > "${REPDIR}/${TOP_ENTITY}_link.rpt"
+analyze -vcs "-sverilog +define+${DEFINE} -f ${SV_FLIST}" > "${REPDIR}/analyze.rpt"
```

Comment:
@sriyerg switched to vcs syntax and this seems to be working fine!
thanks for the tip.